openlibrary/templates/books/custom_carousel.html

```diff
@@ -1,4 +1,4 @@
-$def with(books=[], title="", url="", key="", min_books=1, test=False)
+$def with(books=[], title="", url="", key="", min_books=1, test=False, load_more_url='')
```

Comment:
the load_more_url will be something like `http://openlibrary.org/subjects/fantasy.json?limit=6&offset=12`